client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -142,6 +142,14 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				wp_cache_path: '/wordpress/',
 				wp_cache_readonly: false,
 				wp_cache_writable: false,
+
+				// CDN
+				ossdl_cname: '',
+				ossdl_https: false,
+				ossdl_off_cdn_url: 'http://donnapep.wpsandbox.me',
```

Comment:
I'm guessing this is here for testing purposes until Redux is in place, but just pointing out that this should likely be blank.